test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Csv.Tests.ps1

```diff
@@ -45,27 +45,44 @@ Describe "ConvertTo-Csv" -Tags "CI" {
     }
 
     It "Should output an array of objects" {
-        $result = $testObject | ConvertTo-Csv
+        $result = $testObject | ConvertTo-Csv -IncludeTypeInformation
```

Comment:
I believe we shouldn't add `-IncludeTypeInformation` in tests where we don't check ""#TYPE ".

Reply:
Ah, I miss understood what this one was testing. I will fix it.

Reply:
fixed